dns/resolver.py

```diff
@@ -1079,6 +1079,27 @@ def set_flags(self, flags):
 
         self.flags = flags
 
+    @property
+    def nameservers(self):
+        return self._nameservers
+
+    @nameservers.setter
+    def nameservers(self, nameservers):
+        """
+        :param nameservers: can be a ``str``, ``list``, or None.
+        If it's a ``str``, it will converted to a list.
+        :raise ValueError: if `nameservers` is anything other than \
+        ``str``, ``list``, or None.
+        """
+        if isinstance(nameservers, str):
+            self._nameservers = [nameservers]
```

Comment:
Please note, this behaviour is different from the sugggestion of https://github.com/rthalley/dnspython/issues/403#issuecomment-566656498

Reply:
Read your new comment (https://github.com/rthalley/dnspython/issues/403#issuecomment-571235029). See changes below.